packages/terra-date-input/src/DateInput.jsx

```diff
@@ -5,15 +5,24 @@ import classNamesBind from 'classnames/bind';
 import ThemeContext from 'terra-theme-context';
 import uuidv4 from 'uuid/v4';
 import { injectIntl } from 'react-intl';
-import Input from 'terra-form-input';
 import * as KeyCode from 'keycode-js';
 
+import AccessibleValue from 'terra-time-input/lib/_AccessibleValue';
+import AccessibleInput from 'terra-time-input/lib/_AccessibleInput';
```

Comment:
Both `_AccessibleValue` and `_AccessibleInput` are meant to be private modules in terra-input. Having to add the terra-time-input dependency to be able to use these private modules seems anti-pattern. I don't know of a good and easy option other than to duplicate these for terra-date-input. A long term strategy would be to create a common reusable accessibility toolkit package that can provides some of these reusable modules.

Reply:
I agree it is an anti-pattern. Would it be OK if I left it as-is for now, and then submitted another PR to expose these as two new reusables in terra-core? I feel it is OK to "get away with it" for a few days or a week or so while I make this change. AFIAK end users won't notice.

Reply:
@xenoworf  Yes, we should leave it this way for now until when we see more use cases where we need more of these common modules and we can then plan to create a common place to store them.

Reply:
Logged #1523 to clean up later.